rialto_airflow/harvest/openalex.py

```diff
@@ -119,13 +119,12 @@ def fill_in(snapshot: Snapshot, jsonl_file: Path) -> Path:
             )
             # TODO: consider getting data for more than one DOI at a time
             for row in select_session.execute(stmt):
-                logging.info(f"filling in data for {row.doi}")
```

Comment:
This logging looked like it was going to be excessive in full runs, so removed here as well while working on Dimensions harvesting.